pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -383,8 +392,8 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication {
     }
 
     /**
-     * @dev Manages `account`'s granter condition for action `actionId` in `where`.
-     * Note that pairs can manage themselves, even banning the root, but the root can allow himself back at any time
```

Comment:
I think this comment hints at the undesirable functionality we uncovered. The "pair" management piece seems to imply that a granter has authority to manage all identical granter permissions.